### ISSUE TITLE
Performing string interpolation inside a multiline string inserts incorrect indentation

### ISSUE THREAD

**Issue by bmitc** [sifive/wake#1106]
Inside a Wake multiline string, if you have string interpolation, then it inserts indentation where it shouldn't be. It seems to insert an indentation of 3 spaces. Examples follow.

For the definition:
```wake
export def multiline1 _args =
    println "%
    testing %{str 123}
    %"
```
we get:
```bash
> wake multiline1
testing
Unit
```

But given
```wake
export def multiline2 _args =
    println "%
    testing %{str 123}
    %"
```
we get:
```bash
> wake multiline2
   testing 123
Unit
```

Additionally, the indentation carries over into newlines, as:
```wake
export def multiline3 _args =
    println "%
    testing %{str 123}
    asdf
    %"
```
gives:
```bash
> wake multiline3
   testing 123
   asdf
Unit
```

The only way to get rid of this is to remove some spacing. So:
```wake
export def multiline4 _args =
    println "%
 testing %{str 123}
 asdf
    %"
```
yields:
```bash
> wake multiline4
testing 123
asdf
Unit
```

However, this goes against the documentation in the [quick reference](https://github.com/sifive/wake/blob/master/share/doc/wake/quickref.md), which states that this should actually be an error.

**Comment by ag-eitilt** [comment]
I'm guessing the first example shouldn't have the `%{str 123}`?

**Comment by bmitc** [comment]
Updated description and the end (examples `multiline4` and `multiline5`) to better explain some indentation behavior that addresses a comment in the quick reference.

**Comment by JakeSiFive** [comment]
Yeah this is a bug for sure. Adding string interpolation should not change the indentation removing feature.

**Comment by bmitc** [comment]
Thanks @V-FEXrt for fixing this!
